### PR TITLE
remove ConsensOps from groups

### PR DESCRIPTION
**Pull request by macfarla**
this user does not require elevated access

### CODE REVIEW

**Review by cdivitotawela — APPROVED** [review]
LGTM

File: access-control.yaml [hunk 1]
```diff
@@ -347,10 +347,9 @@ teams:
   - name: besu-contributors
     maintainers:
       - garyschulte
-      - jframe
```
Comment (jframe) [inline access-control.yaml:350]: why did you remove me from the besu-contributors?

Comment (macfarla) [reply access-control.yaml:350]: only so there was one member in the group but I think it's fine for the group to be only maintainers

Comment (macfarla) [reply access-control.yaml:350]: so I reverted this change to your membership :)

**Review by jframe — APPROVED** [review]

**Review by ryjones — APPROVED** [review]
